Pod/Classes/VOKCoreDataManager.m

```diff
@@ -319,46 +296,20 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
         } else {
             returnObject = [self managedObjectOfClass:objectClass inContext:contextOrNil];
             [mapper setInformationFromDictionary:inputDict forManagedObject:returnObject];
+            if (inputKey) {
+                [existingObjectArray addObject:returnObject];
+                [existingUniqueKeys addObject:inputKey];
```

Comment:
Shouldn't existingUniqueKeys already contain inputKey in this case?
You won't get to this branch unless the line 284 is true (`[existingUniqueKeys containsObject:inputKey]`)